app/upgrades.go

```diff
@@ -230,6 +232,41 @@ func removeP8eMemorializeContractFee(ctx sdk.Context, app *App) {
 	}
 }
 
+// removeInactiveValidators unbonds all delegations from inactive validators, triggering their removal from the validator set.
+func removeInactiveValidators(ctx sdk.Context, app *App) {
+	unbondingTimeParam := app.StakingKeeper.GetParams(ctx).UnbondingTime
+	ctx.Logger().Info(fmt.Sprintf("removing any validator that has been inactive (unbonded) for %d hours", int64(unbondingTimeParam.Hours()/24)))
```

Comment:
status message says hours ... but you are dividing the hours param by 24?

Reply:
Good catch!